src/main/java/com/solvd/carina/mobile/android/ProductScreen.java

```diff
@@ -0,0 +1,72 @@
+package com.solvd.carina.mobile.android;
+
+import com.solvd.carina.mobile.base.CartScreenBase;
+import com.solvd.carina.mobile.base.MenuScreenBase;
+import com.solvd.carina.mobile.base.ProductScreenBase;
+import com.zebrunner.carina.utils.factory.DeviceType;
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+
+@DeviceType(pageType = DeviceType.Type.ANDROID_PHONE, parentClass = ProductScreenBase.class)
+public class ProductScreen extends ProductScreenBase {
+
+    @FindBy(xpath = "//android.widget.TextView[@text='PRODUCTS']")
+    private ExtendedWebElement productScreenTitle;
+
+    @FindBy(xpath = "//android.view.ViewGroup[@content-desc=\"test-Menu\"]/android.view.ViewGroup/android.widget.ImageView")
```

Comment:
does it find element with this locator?
```suggestion
    @FindBy(xpath = "//android.view.ViewGroup[@content-desc=\"test-Menu\"]")
```

Reply:
if yes - please update all locators and make them shorter if possible